### PR TITLE
Enforce safety and lintiness

### PR DESCRIPTION
**Pull request by notriddle**

### CODE REVIEW

**Comment by lnicola** [comment]
Should probably update the minimum compiler version to 1.26.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ HTML Sanitization
 =================
 
 [![Crates.IO](https://img.shields.io/crates/v/ammonia.svg)](https://crates.rs/crates/ammonia)
-![Requires rustc 1.24.0](https://img.shields.io/badge/rustc-1.24.0+-green.svg)
+![Requires rustc 1.26.0](https://img.shields.io/badge/rustc-1.24.0+-green.svg)
```
Comment (lnicola) [inline README.md]: Wrong URL.

**Comment by lnicola** [comment]
[1.30](https://doc.rust-lang.org/std/error/trait.Error.html#method.source)

**Comment by notriddle** [comment]
@lnicola https://github.com/SimonSapin/rust-utf8/pull/6

**Comment by lnicola** [comment]
Ouch. Well, `rayon` supports Rust 1.13 and that's pretty annoying. I don't think anyone will complain about requiring 1.30.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-*
+* Bump minimum supported Rust version to 1.26.
```
Comment (lnicola) [inline CHANGELOG.md]: 1.30

File: README.md [hunk 3]
```diff
@@ -2,12 +2,7 @@ HTML Sanitization
 =================
 
 [![Crates.IO](https://img.shields.io/crates/v/ammonia.svg)](https://crates.rs/crates/ammonia)
-![Requires rustc 1.24.0](https://img.shields.io/badge/rustc-1.24.0+-green.svg)
-
-Chat: [Gitter], [Matrix]
-
-[Gitter]: https://gitter.im/rust-ammonia/Lobby
-[Matrix]: https://matrix.to/#/#rust-ammonia:gpmatrix.com
+![Requires rustc 1.30.0](https://img.shields.io/badge/rustc-1.26.0+-green.svg)
```
Comment (lnicola) [inline README.md]: 1.30

**Comment by lnicola** [comment]
bors delegate=notriddle

**Comment by notriddle** [comment]
bors r-

bors r=notriddle,lnicola

**Comment by lnicola** [comment]
Huh?

**Comment by lnicola** [comment]
bors r=notriddle,lnicola

**Pull request merged**